src/widgets/BaseWindow.cpp

```diff
@@ -563,9 +563,9 @@ void BaseWindow::resizeEvent(QResizeEvent *)
             });
         });
     }
-#endif
 
     this->calcButtonsSizes();
+#endif
```

Comment:
Why was this moved? Shouldn't platforms other than Windows still recalculate the button sizes?

Reply:
said buttons are part of custom frame, windows only. All other calls to calcButtonsSizes() are already ifdef'ed